libraries/Wire/Wire.h

```diff
@@ -28,9 +28,7 @@
 
 #include "RingBuffer.h"
 
-#define BUFFER_LENGTH 32
```

Comment:
The `SERIAL_BUFFER_SIZE` macro is provided by "RingBuffer.h", which can be used for the original purpose of this macro.
Maybe it'd be better to do `#define BUFFER_LENGTH SERIAL_BUFFER_SIZE `? Which library uses this macro that is not specific to AVR?